### PR TITLE
JumpingBytes.PureSync version 7.1.2

### PR DESCRIPTION
**Pull request by CanePlayz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99220)

### CODE REVIEW

File: manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JumpingBytes.PureSync
```
Comment (mdanish-kh) [inline manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.installer.yaml:4]:
```suggestion
PackageIdentifier: JumpingBytes.PureSync.Personal
```
As the package has two editions, a personal and a professional one, I would prefer if we disambiguate the two in the PackageIdentifier.

File: manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JumpingBytes.PureSync
```
Comment (mdanish-kh) [inline manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: JumpingBytes.PureSync.Personal
```
As the package has two editions, a personal and a professional one, I would prefer if we disambiguate the two in the PackageIdentifier.

File: manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: JumpingBytes.PureSync
```
Comment (mdanish-kh) [inline manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.yaml:4]:
```suggestion
PackageIdentifier: JumpingBytes.PureSync.Personal
```
As the package has two editions, a personal and a professional one, I would prefer if we disambiguate the two in the PackageIdentifier.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to modify the directory structure to match the new PackageIdentifier. `manifests/j/JumpingBytes/PureSync/7.1.2` should be restructured to `manifests/j/JumpingBytes/PureSync/Personal/7.1.2`. Subsequently, the file names would also require re-naming to match the new Package ID. (JumpingBytes.PureSync.installer.yaml -> JumpingBytes.PureSync.Personal.installer.yaml and so on)

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by CanePlayz** [comment]
Thanks, so much stuff I didn't know about 😅 I thought I had changed the description from the default one, but apparently, I hadn't.

Finally, the GitHub extensions in VS Code are coming in handy ;)

image

**Comment by mdanish-kh** [comment]
> Thanks, so much stuff I didn't know about 😅 I thought I had changed the description from the default one, but apparently, I hadn't.

No issues.. :) Getting started is the hard part with the manifests submissions, you'll get a hang of it once you submit a couple of them; and the mods would be there to help you with your submissions.

**Review by mdanish-kh — APPROVED** [review]

**Comment by CanePlayz** [comment]
> > Thanks, so much stuff I didn't know about 😅 I thought I had changed the description from the default one, but apparently, I hadn't.
> 
> No issues.. :) Getting started is the hard part with the manifests submissions, you'll get a hang of it once you submit a couple of them; and the mods would be there to help you with your submissions.

Actually, there's one more thing I need to fix I think, and that would be indentation. My YAML extension automatically added indentation here:

```
Installers:
  - InstallerUrl:
```

I couldn't find that indentation in other manifests, so should I remove it or won't that matter?

**Comment by mdanish-kh** [comment]
> I couldn't find that indentation in other manifests, so should I remove it or won't that matter?

The indentation won't matter in this case

>    VS Code highlights the `PackageVersion` because it expects a string based on the YAML schema. But it doesn't do that with the `ManifestVersion`, so what's the difference between those two? How does it behave for you?

You could add wrap it with double-quotes but it won't make a difference here too. As long as the manifest passes `winget validate --manifest <path_to_manifest>`, you can be almost sure that there isn't any syntax non-sense messing with the manifest.

**Comment by CanePlayz** [comment]
> The indentation won't matter in this case

Good 👍

> You could add wrap it with double-quotes but it won't make a difference here too. As long as the manifest passes winget validate --manifest <path_to_manifest>, you can be almost sure that there isn't any syntax non-sense messing with the manifest.

Good as well, then I can get rid of the syntax highlighting without having to worry about the validity of the manifest.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**